hawkular-swarm-agents/hawkular-swarm-agent-dists/hawkular-swarm-agent-dist/src/main/resources/hawkular-swarm-agent-config.xml

```diff
@@ -1036,7 +1036,7 @@
 
       <managed-servers>
         <remote-dmr enabled="${hawkular.dmr.enabled:true}"
-                    name="Wildfly"
+                    name="WildFly"

```

Comment:
this doesn't have anything to do with this PR.... ah, what the heck, its Tuesday.